lib/components/form/styled.ts

```diff
@@ -230,3 +231,78 @@ export const StyledLocationField = styled(LocationField)`
     }
   }
 `
+
+export const advancedPanelClassName = 'advanced-panel'
+export const mainPanelClassName = 'main-panel'
+export const transitionDuration = prefersReducedMotion ? 0 : 175
+export const transitionDelay = prefersReducedMotion ? 0 : 300
```

Comment:
Can we tighten this up? 150ms-200ms total animation time feels snappier to me. The delay I'm guessing is for the save button? That feels perfect

Reply:
The transition animation is only `175`, which is right in that window! And yes, the 300 is for the delay

Reply:
Are you ok with `100`? I've tweaked it and it feels much snappier

Reply:
`100` for a full panel moving in and out is really fast. [`175` is already aggressive](https://www.nngroup.com/articles/animation-duration/#:~:text=When%20animation%20involves%20substantial%20screen%20changes%2C%20such%20as%20when%20a%20modal%20window%20moves%20into%20view%2C%20a%20duration%20of%20200%E2%80%93300%20ms%20can%20be%20appropriate.) for how much content is transitioning.